dev/com.ibm.ws.microprofile.config.1.4/test/src/com/ibm/ws/microprofile/config14/test/ConversionTest.java

```diff
@@ -46,6 +51,32 @@ public void testBadCharUnicode() {
         assertBadConversion("testBadCharUnicode", raw, char.class);
     }
 
+    @Test
+    public void testList() {
```

Comment:
These new unit tests should be in the 1.2 project since that is where the fix was made